docs/use/Set-Up-DeFi-Tools/index.md

```diff
@@ -0,0 +1,62 @@
+
+### How to install Metamask in your browser
+1. Go to https://metamask.io/download.html.
+2. Choose the browser you want to use.
+3. You will be directed to corresponding web store.
+4. Follow the steps there to add the extension to your browser.
+5. After setting up your wallet, Metamask should pop-up as a Fox icon ![](https://storage.googleapis.com/centrifuge-hackmd/upload_7dd07df8a7fc7d1b29743ec9535dbba7.png =20x) in your browser bar.
+
+### How to use Metamask with Tinlake
+- Connect your account.
+- Click on Connect (top right corner).
+- Choose the Metamask or the wallet you would like to connect.
+![](https://storage.googleapis.com/centrifuge-hackmd/upload_7ce191ec0d5455f0228b8e9779c54719.png)
+
+## Set up a Ledger Hardware Wallet
+### How to use Metamask with a hardware wallet, e.g. Ledger?
+A hardware wallet stores the user's private keys in a secure hardware device. To use an address/wallet stored on a hard ware wallet a user needs to be in physical posession of this hardware wallet. Popolar hardware wallets are e.g [Ledger](https://www.ledger.com/) or [Trezor](https://trezor.io/).
+
+#### Set up the Ledger
+Download the [Ledger Live App](https://www.ledger.com/ledger-live/) and follow the steps in the App also described [here]( https://support.ledger.com/hc/en-us/articles/360018784134) to set up you Ledger.
+
+### Install Ethereum App on Ledger Device
+1. Open the Manager in Ledger Live.
+2. Connect and unlock your device.
+3. If asked, allow the manager on your device.
+4. Find Ethereum in the app catalog.
+5. Click the Install button of the app.
+    - An installation window appears
+    - Your device will display `Processing...`
+    - The app installation is confirmed
+
+See also [Install Ethereum App on your Ledger](https://support.ledger.com/hc/en-us/articles/360009576554-Ethereum-ETH-).
+
+### Configure Ethereum App 
+
+1. Open the Ethereum App Settings.
+2. Change Contract Data to `Yes`.
+3. [OPTIONAL] For older versions of Ledger, make sure Browser Support is set to `Yes`. Newer Ledger Versions are supporting it by default. 
+
+### Connect Hardware Wallet in Metamask
+1. Open Metamask by clicking on the fox icon.
+2. Make sure to select the correct network (`Main Ethereum Network` or `Kovan Test Network`) at the top.
+3. Click on the circle in the top right corner.
+4. Click on `Connect Hardware Wallet`.
+5. A new modal opens. Select `Ledger` and click `Connect`.
+6. Select address you would like to use. Your Ledger already entails several addresses you can chose from (autogenerated for you).
+
+### Confirm transactions with your hardware wallet (Ledger/Trezor)
+When your Metamask account is connected to your hardware wallet then you need to confirm each transaction with the ledger. 
```

Comment:
When your MetaMask account is connected to the hardware wallet you need to confirm each transaction with Ledger.